psl/psl-core/src/validate/validation_pipeline/validations.rs

```diff
@@ -97,6 +98,7 @@ pub(super) fn validate(ctx: &mut Context<'_>) {
             relation_fields::ignored_related_model(field, ctx);
             relation_fields::referential_actions(field, ctx);
             relation_fields::map(field, ctx);
+            relation_indexes::validate_missing_relation_indexes(field, ctx);
```

Comment:
Do we really need our own module for this? Could this fit to the `relation_fields` model?

Reply:
I don't have a strong opinion about this